### PR TITLE
Fix improper dates for event submit

### PR DESCRIPTION
**Pull request by crawsoir**
- Prevent form from submitting when user clicks Submit, if start date is after end date
- Throw an alert when this happens

image

### CODE REVIEW

File: utmap-client/src/components/pages/CreateEventPage.js [hunk 1]
```diff
@@ -66,13 +66,18 @@ function CreateEventPage({onClose, addEvent}) {
...
 		endDate.setSeconds(0,0);
-		const eventForm = {
-			title, 
-			startDate: startDate.toLocaleString('en-US', { timeZone: 'America/New_York' }), 
-			endDate: endDate.toLocaleString('en-US', { timeZone: 'America/New_York' }),
-			location, sublocation, description};
-		addEvent(eventForm);
-		onClose();
+		
+		if (startDate > endDate){
```
Comment (RiddleMeDoo) [inline utmap-client/src/components/pages/CreateEventPage.js:70]: I don't think a simple comparison will work. 
Eg.
```JavaScript
startDate = "10/5/2021, 6:00:00 PM";
endDate = "1/5/2021, 6:00:00 PM";
startDate > endDate  == True
```

Comment (RiddleMeDoo) [reply utmap-client/src/components/pages/CreateEventPage.js:70]: Do `Date.parse(startDate) > Date.parse(endDate)` to convert them into a number instead  
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

Comment (crawsoir) [reply utmap-client/src/components/pages/CreateEventPage.js:70]: It does seem to work? Unless it's bad form to not do a number conversion first

image

Comment (RiddleMeDoo) [reply utmap-client/src/components/pages/CreateEventPage.js:70]: Ahhh my bad I thought you converted to string and then compared.

**Review by RiddleMeDoo — APPROVED** [review]

**Review by ShahmeerShahid — APPROVED** [review]

**Pull request merged**